### ISSUE TITLE
[Feature request] Option for Larger icons

### ISSUE THREAD

**Issue by GammaMetroid** [hexabits/nifskope#59]
Even using the large icons option, they are still rather small on my screen. My screen resolution is 3840 x 2160 and I am using Windows' scaling option to scale things by 300%.

![nifskope_scale](https://github.com/hexabits/nifskope/assets/101038501/55c1721f-5a55-4006-96c2-094a601df3fa)

**Comment by hexabits** [comment]
@GammaMetroid there is. 
![image](https://github.com/hexabits/nifskope/assets/170077/81c17ada-28d5-41c0-b649-2f16a3af50cc)

**Comment by hexabits** [comment]
Are you certain this is using the Large Icons option?  Did you hit Save/Apply and restart?

**Comment by ja231c** [comment]
The size of those icons does not change with the Windows scaling option at all, stuff in that toolbar in general has problems with scaling, always had.
The new Dev 9 LOD slider also has problems with scaling, the gaps between the LOD levels are tiny above 100% scaling, from what I can see on that screenshot it seems at 300% it's not even visible at all, even the "LOD" text is cut off.

Also, #50 is somewhat related.

**Comment by gavrant** [comment]
@GammaMetroid @ja231c Guys, the size of the toolbar icons does not change at all for you if you check or uncheck the settings' option on the screenshot below and click "Save"?
![nifsklargeicons](https://github.com/hexabits/nifskope/assets/17292867/27a60384-6927-439f-8ea9-9918a20bb28a)

**Comment by gavrant** [comment]
OK, it seems the toolbars specifically are not affected by Windows scaling settings, while everything else is (EDIT: well, most of the rest is):
<details>
<summary>Screenshot</summary>

![niffontscale](https://github.com/hexabits/nifskope/assets/17292867/ac854443-eb22-4eb9-8e73-deff61c86d11)

</details>
@hexabits , I can make a run at the issue if you don't mind.

**Comment by GammaMetroid** [comment]
Yes, I definitely have the large icons options checked. Here is what it looks like without. You may notice that the file path down at the bottom is also tiny.
![nifskope_scale_small](https://github.com/hexabits/nifskope/assets/101038501/e80bb70d-c2c8-4f66-882a-cc4c48feb33a)
